etc/kayobe/ansible/rabbitmq-reset.yml

```diff
@@ -68,4 +68,4 @@
     # The following services use RabbitMQ.
     - name: Restart OpenStack services
       shell: >-
-        systemctl -a | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova|octavia)' | awk '{ print $1 }' | xargs systemctl restart
+        systemctl list-units --type=service | egrep '(barbican|blazar|cinder|cloudkitty|designate|heat|ironic|keystone|magnum|manila|neutron|nova|octavia)' | egrep -v ceph | awk '{ print $NF }' | xargs systemctl restart
```

Comment:
I see this is to match the description. A bit dangerous if this changes in kolla-ansible.

Reply:
New approach, splitting every word out to a new line and selecting unique entries with better regex matching (must start with `kolla-`, end with `-container.service`

Reply:
Will this now pick up each container twice, from the name and description? Could there be any issues from calling the restart twice in quick succession?

Reply:
`systemctl list-units` supports passing a pattern in, can we do that instead of the grep? https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#list-units%20PATTERN%E2%80%A6

Reply:
I added `| uniq` to stop it duplicating lines.

I don't think the systemctl pattern really helps. Grep is more versatile here